### ISSUE TITLE
Avogadro 2 keep fixed selected atoms option doesn't work

### ISSUE THREAD

**Issue by martino2650** [OpenChemistry/avogadrolibs#1684]
**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: (e.g. 1.90.0-316-g6d14770)
 - Qt: (e.g., 5.9.5)

**Desktop version: (please complete the following information):**
 - OS: [e.g. MacOS]
 - Version [e.g. 10.12.4]
 - Compiler (if you built from source)

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**
Add any other context about the problem here. **Please consider uploading or linking test files.**

**Comment by ghutchis** [comment]
You haven't given much information on what you mean. What version are you using? What OS? What particular commands are you using that "don't work?"

**Comment by martino2650** [comment]
You're right.

I'am using Avogadro 2 (version 1.99.0) on Windows 10.

I draw the structure of interest; then I select with the selection tool the
atoms that I want to keep fixed; I click on Extensions -> Fix selected
Atoms; I run the optimization.

This procedure works on Avogadro 1 (version 1.20n).

Il mer 3 lug 2024, 17:19 Geoff Hutchison ***@***.***> ha
scritto:

**Comment by ghutchis** [comment]
Just to be clear, you're using the "Optimize Geometry" and not Open Babel, right? Do you know what force field is selected?

Thanks.

**Comment by martino2650** [comment]
Yes, I'm using "Optimize Geometry", not Open Babel. I don't know which
Force Field I'm using because the program doesn't let me to choose.

Il gio 4 lug 2024, 17:39 Geoff Hutchison ***@***.***> ha
scritto:

**Comment by ghutchis** [comment]
This works for me on both Mac and Windows with recent builds. Can you try a new nightly build?

https://two.avogadro.cc/install/

You can check the force field for "Optimize Geometry" under Calculate ⇒ Configure… (or change it as you wish)

**Comment by matterhorn103** [comment]
Works for me on Linux too with the Flatpak. Closing for now, happy to reopen if the feature isn't working for someone else.
